exec_test.go

```diff
@@ -35,3 +36,43 @@ func ExampleCommandExists() {
 	// true
 	// false
 }
+
+func BenchmarkCommandExists(b *testing.B) {
+	cases := []struct {
+		Name    string
+		Command string
+	}{
+		{"cmd1", "go"},
+		{"cmd2", "foo"},
+		{"cmd3", "bar"},
+		{"cmd4", "foo"},
+	}
+	for _, bc := range cases {
+		b.Run(bc.Name, func(b *testing.B) {
+			b.ResetTimer()
+			for i := 0; i < b.N; i++ {
+				u.CommandExists(bc.Command)
+			}
+		})
+		b.Run(bc.Name+"-parallel", func(b *testing.B) {
+			b.ResetTimer()
+			b.RunParallel(func(pb *testing.PB) {
+				for pb.Next() {
+					u.CommandExists(bc.Command)
+				}
+			})
+		})
+	}
+}
+
+func BenchmarkSafeExec(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		u.SafeExec(exec.Command("sh", "-c", "echo stdout; echo 1>&2 stderr; exit 1"))
```

Comment:
`sh -c echo` is slow, we should find something like `true` or a very very fast binary

Reply:
@moul I still don't have any example for this hmm?